client/test/loadGameJSON.ts

```diff
@@ -185,8 +185,14 @@ export default function loadGameJSON(gameJSON: JSONGame): State {
     replay: { ...state.replay, states },
     cardIdentities: [],
 
+    // TODO the test data has no notes,
```

Comment:
probably can delete this, unless you actually intend to write note tests in the future